tox.ini

```diff
@@ -14,7 +14,7 @@ deps = poetry >= 1.0.5
 commands =
     poetry install
     poetry env info
-    poetry run py.test -v test
+    poetry run py.test -v test -k "not integration"
```

Comment:
this is clever. I had assumed these would get split up into `test/unit` and `test/integration`, but clearly this works.